nat-lab/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[project]
+name = "nat-lab"
+
+[tool.black]
+preview = true
+skip_magic_trailing_comma = true
```

Comment:
Why is `skip_magic_trailing_comma` needed?

Reply:
```
some_list = [pytest.mark.linux_native, pytest.mark.windows,]
```
`skip_magic_trailing_comma = true` removes trailing comma after `pytest.mark.windows` and doesn't break list line if it fits to one line

if `skip_magic_trailing_comma = false`, this would happen:
```
some_list = [
    pytest.mark.linux_native,
    pytest.mark.windows,
]
```

Reply:
My understanding is that if `skip_magic_trailing_comma = false` then when you write:
```
some_list = [pytest.mark.linux_native, pytest.mark.windows,]
```
it will do:
```
some_list = [
    pytest.mark.linux_native, 
    pytest.mark.windows,
]
```
and if you write:
```
some_list = [pytest.mark.linux_native, pytest.mark.windows]
```
it will stay this way. 
I think it is a good default.
Maybe it was useful to enable this option to sqash some places to one liners where they fit (I actually like this a lot) but now you can check if something changes if you disable it. My guess is that black should not change the current code and it will let you decide easily influence how it behaves in different places: You want items in separate lines because it is actually a list that can grow then add a tralling comma. If you want one liner do not add it and it should stay this way.

Reply:
alrighty, removed, but didn't reset changes that were already made if it's ok with you :smile:

Reply:
Yes, that was my point. I wanted them to stay :)